maps/README.txt

```diff
@@ -0,0 +1,6 @@
+The goal of this project is to create a sample interactive ad creative for SIMID. 
```

Comment:
Looks like you accidentally added .README.txt.swp, be sure to remove

You can add the line `*.swp` to your .gitignore file and that should prevent accidentally adding swp files

Reply:
Deleted.